mozilla/activity-stream
# fix(snippets): Don't block snippets on onboardingFinished

Manual test:
- onboarding should show if snippets are enabled in the prefs panel, and onboarding has notifications to show
- onboarding should be visually hidden if snippets are turned off in the prefs panel
- onboarding should show up on the next new tab if snippets are turned back on in the prefs panel

- while onboarding is showing, call `gSnippetsMap.disableOnboarding()` in your console. Nothing should visually happen. on the next new tab, onboarding should be hidden and the default pref for onboarding.notifications.finished should be true.
- restart your browser. onboarding should be shown again
